geometry/create-and-edit-geometries/src/main/java/com/esri/samples/create_and_edit_geometries/CreateAndEditGeometriesController.java

```diff
@@ -0,0 +1,440 @@
+        ));
+      } else {
+        // when the geometry editor is not started (and so the geometry is null) enable all geometry buttons
+        pointButton.disableProperty().unbind();
+        pointButton.setDisable(false);
+        multipointButton.disableProperty().unbind();
+        multipointButton.setDisable(false);
+        polylineButton.disableProperty().unbind();
+        polylineButton.setDisable(false);
+        polygonButton.disableProperty().unbind();
+        polygonButton.setDisable(false);
+
+        // disable combo box when geometry editor is not started
+        toolComboBox.disableProperty().unbind();
+        toolComboBox.setDisable(true);
+      }
+    });
+
+    // bidirectionally bind the geometry editor tool to the tool selected in the combo box
+    toolComboBox.valueProperty().bindBidirectional(geometryEditor.toolProperty());
+
+    // bind the disable property of the undo & redo buttons to the geometry editor's can undo & can redo properties
+    undoButton.disableProperty().bind(geometryEditor.canUndoProperty().not());
+    redoButton.disableProperty().bind(geometryEditor.canRedoProperty().not());
+
+    geometryEditor.selectedElementProperty().addListener(((observable, oldValue, newValue) -> {
+      if (newValue != null) {
+        // if an element is selected in the geometry editor
+        // bind the disable property of the delete selected element button to the selected element's can delete property
+        deleteSelectedElementButton.disableProperty().bind(geometryEditor.getSelectedElement().canDeleteProperty().not());
+      } else {
+        // if no element is selected in the geometry editor disable the delete selected element button
+        deleteSelectedElementButton.disableProperty().unbind();
+        deleteSelectedElementButton.setDisable(true);
+      }
+    }));
+
+    // bind the disable property of the stop and save button to the geometry editor's started and can undo properties
+    stopAndSaveButton.disableProperty().bind(Bindings.or(geometryEditor.startedProperty().not(), geometryEditor.canUndoProperty().not()));
+
+    // bind the disable property of the stop and discard button to the geometry editor's started property
+    stopAndDiscardButton.disableProperty().bind(geometryEditor.startedProperty().not());
+
+    // bind the disable property of the delete all geometries button to the geometry editor's started property
+    // and to whether there are any graphics in the graphics overlay
+    deleteAllGeometriesButton.disableProperty().bind(Bindings.or(
+      geometryEditor.startedProperty(), Bindings.createBooleanBinding(() -> graphicsOverlay.getGraphics().isEmpty())));
+
+    // allow user to immediately select and edit graphics
+    mapView.setOnMouseClicked(e -> {
+      if (e.getButton() == MouseButton.PRIMARY && e.isStillSincePress()){
+      graphicsOverlay.clearSelection();
+      Point2D mapViewPoint = new Point2D(e.getX(), e.getY());
```

Comment:
```suggestion
      if (e.getButton() == MouseButton.PRIMARY && e.isStillSincePress()) {
      graphicsOverlay.clearSelection();
      var mapViewPoint = new Point2D(e.getX(), e.getY());
```
Also check the indentation here, I think the if statement body needs indenting